app/experimenter/experiments/constants/nimbus.py

```diff
@@ -59,10 +59,22 @@ class NimbusTargetingConfig:
         first_run=TARGETING_FIRST_RUN.targeting
     ),
     desktop_telemetry=(
-        "STARTS_WITH(environment.settings.locale, 'en') "
-        "AND payload.info.profile_subsession_counter = 1"
```

Comment:
This line didn't have a trailing space before. Not sure if that had an impact or is important?